frontend/src/theme/components/Tile.ts

```diff
@@ -0,0 +1,43 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export const Tile: ComponentMultiStyleConfig = {
+  parts: ['container', 'title', 'icon', 'subtitle'],
+  baseStyle: {
+    container: {
+      borderRadius: '0.25rem',
+      padding: '1.5rem',
+      _active: {
+        bgColor: 'primary.200',
+        border: '3px solid var(--chakra-colors-primary-500)',
```

Comment:
Since the box sizing has not changed, this causes the component to "squish" when activated.

Reply:
fixed using the `boxShadow` trick!